### ISSUE TITLE
Setting debug level prints to console

### ISSUE THREAD

**Issue by politeauthority** [quandyfactory/dicttoxml#58]
Is it possible to remove the "Debug mode is off." print statement in the console on line 42/ 49 or **dicttoxml.py**. I really like this module, but pride myself on not having random print statements to the console. I'm using a ton of other modules in conjunction with dicttoxml and this print statement just ends up being confusing and unnecessary. I can make a PR for this if it helps at all. Thanks!

**Comment by andreibancioiu** [comment]
```
import logging
import dicttoxml

dicttoxml.LOG.setLevel(logging.ERROR)
```

**Comment by chintanvadgama** [comment]
Hi,

Thanks a lot for the fix!

**Comment by quandyfactory** [comment]
This issue is fixed in version 1.7.7.
